cluster/charts/rook-ceph-cluster/values.yaml

```diff
@@ -270,11 +287,130 @@ cephClusterSpec:
       status:
         disabled: false
         interval: 60s
-    # Change pod liveness probe, it works for all mon,mgr,osd daemons
+    # Change pod liveness probe, it works for all mon, mgr, and osd pods.
     livenessProbe:
       mon:
         disabled: false
       mgr:
         disabled: false
       osd:
         disabled: false
+
+ingress:
```

Comment:
Above this line is entirely indentation fixes, the indentation of the commented out sections was inconsistent and if this helm chart is going to start being published we should fix it, let me know if you'd rather this in a separate PR.